pkg/kubelet/certificate/certificate_store.go

```diff
@@ -195,6 +195,9 @@ func (s *fileStore) Update(certData, keyData []byte) (*tls.Certificate, error) {
 	ts := time.Now().Format("2006-01-02-15-04-05")
 	pemFilename := s.filename(ts)
 
+	if err := os.MkdirAll(s.certDirectory, 0755); err != nil {
```

Comment:
Hmm. Maybe leave it as is then. Probably better to change it everywhere in a different PR if we care about this.